README.md

```diff
@@ -1,5 +1,40 @@
-# lighthouse 
-Controller to facilitate DNS discovery in multi-cluster connected environments (proof-of-concept state)
+# Lighthouse 
+Lighthouse provides DNS discovery to clusters connected by [Submariner](https://github.com/submariner-io/submariner) in multi-cluster connected environments. The solution aims to compatible with any CNI-compatible cluster network provider.
```

Comment:
DNS discovery to **Kubernetes** clusters

Reply:
Done.